package_build_list_host_darwin.json

```diff
@@ -67,6 +67,7 @@
         "mikkelsen-1.0.0.4-mac": "package-system/mikkelsen-mac",
         "mikkelsen-1.0.0.4-ios": "package-system/mikkelsen-ios",
         "poly2tri-7f0487a-rev1-mac": "package-system/poly2tri-mac",
+		"RapidJSON-1.1.0-multiplatform": "package-system/RapidJSON-multiplatform",
```

Comment:
The alignment is off in this file

Reply:
Oh, I was working on my laptop for the first time in forever, which I guess hasn't been set up properly. Thanks.